public/index.html

```diff
@@ -70,6 +71,27 @@
                         <button type="button" class="btn btn-block btn-primary" data-toggle="modal" data-target="#collaboration_menu">
                             Open collaboration menu
                         </button>
+
+                        <!-- Collapse -->
+                        <!-- Inspired by: https://disjfa.github.io/bootstrap-tricks/card-collapse-tricks/ -->
+                        <button type="button" class="btn btn-block btn-primary collapsed" data-toggle="collapse" data-target="#visualization_collapse">
+                            <div class="d-flex flex-row w-100"><div class="w-100">Visualization adjustments</div> <i class="collapse_rotate fa fa-chevron-down pull-right pt-1"></i></div>
+                        </button>
+                        <div class="m-2">
+                            <div class="collapse card" id="visualization_collapse">
+                                <div class="card-body">
+                                    <h6 class="card-subtitle text-muted">Brightness</h6>
+                                    <span class="m-2">
+                                        -1 <input id="brightness_slider" type="text" data-slider-min="-1" data-slider-max="1" data-slider-step="0.1" data-slider-value="0"/> &nbsp;1
+                                    </span>
+                                    <h6 class="card-subtitle mt-2 text-muted">Contrast</h6>
+                                    <span class="m-2">
+                                        -1 <input id="contrast_slider" type="text" data-slider-min="-1" data-slider-max="1" data-slider-step="0.1" data-slider-value="0"/> &nbsp;1
```

Comment:
Might want to add a button for quickly resetting all sliders to 0 in the future.